src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -571,7 +571,7 @@ static String getResource(String key) {
                     "The client key used to access the Key Vault where the column encryption master key is stored."},
             {"R_ADALMissing", "Failed to load ADAL4J Java library for performing {0} authentication."},
             {"R_DLLandADALMissing",
-                    "Failed to load both sqljdbc_auth.dll and ADAL4J Java library for performing {0} authentication. Please install one of them to proceed."},
+                    "Failed to load both mssql-jdbc_auth and ADAL4J Java library for performing {0} authentication. Please install one of them to proceed."},
```

Comment:
Should we spell out the full filename for mssql-jdbc_auth so it is clear to users what file they need when they encounter this error?